hathor/p2p/sync_v2/manager.py

```diff
@@ -388,7 +389,7 @@ def send_get_tips(self) -> None:
         self.send_message(ProtocolMessages.GET_TIPS)
         self._receiving_tips = []
 
-    def handle_get_tips(self, payload: str) -> None:
+    def handle_get_tips(self, _payload: str) -> None:
```

Comment:
What's the benefit of prefixing the argument with `_`, the only reason I could see is making it so we don't want it to be used as a keyword argument, but for that we can make it positional only with `handle_get_tips(self, payload: str, /)` which is available since Python 3.8 I think.

This comment applies to all the renames of `payload` to `_payload`.

Reply:
This is just convention since this argument is not used in this method, so that intention is made clear by using `_payload`. It could even be just `_`, but in this case I prefer the named unused parameter, also for clarity

Reply:
Oooh, got it, if it's unused but needs to exist it makes sense, but can't we just remove them instead?

Reply:
Oh, because of the signature of the dict that stores the handlers we might need the argument. I'm OK using `_payload` then.

Reply:
Exactly, we can't because it's used in the commands dict:

```python
def get_cmd_dict(self) -> dict[ProtocolMessages, Callable[[str], None]]:
    ...
```

So the callable needs the string argument. The other option would be to remove the argument from the method and then use a lambda in the cmd_dict, `lambda _: self.handle_get_tips()`, so it conforms to the callable type

Reply:
I've never seen this convention but I'm ok with it as well.